tests/test_calendar_two_lane_schema.py

```
@@ -174,6 +175,149 @@ def test_importance_check_constraint_rejects_integer(store: SQLiteEngineStore) -
+
+    events = store.list_events_in_range(date_from=start, date_to=end)
+
+    assert len(events) == 1
+    assert events[0].timestamp == start
+    assert events[0].event_id == "tradingeconomics:date-only"
+
+
+def test_latest_released_event_prefers_newest_before_importance(
+    store: SQLiteEngineStore,
+) -> None:
+    older = datetime(2026, 4, 22, 12, tzinfo=timezone.utc)
```

Ruff (0.15.10)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)